### PR TITLE
Release 1.6.0

### PR DESCRIPTION
**Pull request by CyberDem0n**
* Implement missing tests and do a few minor fixes
* Bump version to 1.6.0
* Update release notes

### CODE REVIEW

**Comment by avaczi** [comment]
There are a number of places in the `.rst` files where a single backtick is used instead of double bacticks

File: docs/releases.rst [hunk 1]
```diff
@@ -3,6 +3,118 @@
...
+
+**New features**
+
+- Psycopg2 was removed from requirements and must be installed independently (Alexander Kukushkin)
+
+  Starting from 2.8.0 `psycopg2` was split into two different packages, `psycopg2`, and `psycopg2-binary`, which could be installed at the same time into the same place on the filesystem. In order to decrease dependency hell problem, we let a user choose how to install it. There are a few options available, please consult the :ref:`documentation <psycopg2_install_options>`.
+
+- Compatibility with PostgreSQL 12 (Alexander Kukushkin)
+
+  Starting from PostgreSQL 12 there is no `recovery.conf` anymore and all former recovery parameters are converted into `GUC`. In order to protect from ``ALTER SYSTEM SET primary_conninfo`` or similar, Patroni will parse `postgresql.auto.conf` and remove all standby and recovery parameters from there. Patroni config remains backward compatible. For example despite `restore_command` being a GUC, one can still specify it in the ``postgresql.recovery_conf.restore_command`` section and Patroni will write it into `postgresql.conf` for PostgreSQL 12.
```
Comment (avaczi) [inline docs/releases.rst]: Maybe a link from GUC to https://www.enterprisedb.com/blog/what-is-a-guc-variable could help some readers

**Comment by avaczi** [comment]
:+1:

**Comment by CyberDem0n** [comment]
:+1:

**Pull request merged**